### PR TITLE
Update for operate-first timeout issue

### PR DESCRIPTION
**Pull request by lmzuccarelli**
This PR addresses the issues with operate-first taskrun timeouts

- The taskrun (default to 1 hour) timeout error, is causing the build to fail on operate-first
- This could be due to bandwidth throttling on the cluster itself.
- I checked that all kvm-device-plugins were deployed to each node via the daemonset - 26 pods in total (I doubt that this would cause the issue)
- The taskrun timeout setting can be updated in a configmap in the tekton-pipeline namespace - we don't have cluster-admin access so this cant be updated
- I have added a 10G PV to allow for the tasks to be more granular
- The tasks are
  - scos-cosa-copy
  - scos-cosa-init
  - scos-cosa-build
  - scos-cosa-buildextend
  - scos-cosa-test
- I created a new **pipelinerun** task with the timeout override values
- I have left the original pipeline and build task intact
- We will need to do a deep dive into the root cause (as the full build takes about 20 minutes on my local kind cluster)

### CODE REVIEW

**Comment by LorbusChris** [comment]
@lmzuccarelli to me it looks like this would break running the pipeline locally, as it removes the kvm device plugin DaemonSet. Could we instead sync what's here with what's in `apps` and just exclude those resources from the `operate-first` env overlay? (we could also create another env overlay for plain/new OpenShift clusters that also includes them, like in the local env, so people won't have to dig into `apps` to deploy all of this)

**Comment by lmzuccarelli** [comment]
Updated the timeout to 3hrs  - operate-first pipelinerun success

```
tkn pr describe scos-pipelinerun-all
I0831 14:47:32.142876 2784441 request.go:665] Waited for 1.198084847s due to client-side throttling, not priority and fairness, request: GET:https://api.smaug.na.operate-first.cloud:6443/apis/controlplane.operator.openshift.io/v1alpha1
Name:              scos-pipelinerun-all
Namespace:         okd-team
Pipeline Ref:      scos-all
Service Account:   pipeline
Labels:
 tekton.dev/pipeline=scos-all

🌡️  Status

STARTED      DURATION   STATUS
1 hour ago   1h11m8s    Succeeded

⚓ Params

 NAME        VALUE
 ∙ version   4.12

📂 Workspaces

 NAME                 SUB PATH   WORKSPACE BINDING
 ∙ shared-workspace   ---        VolumeClaimTemplate

🗂  Taskruns

 NAME                                                 TASK NAME               STARTED          DURATION   STATUS
 ∙ scos-pipelinerun-all-scos-cosa-test-q6n98          scos-cosa-test          6 minutes ago    2m41s      Succeeded
 ∙ scos-pipelinerun-all-scos-cosa-buildextend-dztkt   scos-cosa-buildextend   33 minutes ago   27m3s      Succeeded
 ∙ scos-pipelinerun-all-scos-cosa-build-8q9dg         scos-cosa-build         1 hour ago       40m38s     Succeeded
 ∙ scos-pipelinerun-all-scos-cosa-init-twv9h          scos-cosa-init          1 hour ago       29s        Succeeded
 ∙ scos-pipelinerun-all-scos-cosa-copy-vtgrw          scos-cosa-copy          1 hour ago       17s        Succeeded

```

File: README.md [hunk 1]
```diff
@@ -82,39 +83,44 @@ tkn pipeline start scos-build \
...
-│       │       └── view.yaml
-│       └── tasks
-│           └── base
-│               ├── git-clone.yaml
-│               ├── kustomization.yaml
-│               └── scos-build.yaml
+│   └── tekton
+│       ├── daemonsets
+│       │   ├── kustomization.yaml
+│       │   └── kvm-dev-plg-ds.yaml
```
Comment (sherine-k) [inline README.md]:
```suggestion
│       ├── daemonsets
│       │   └── base
│       │       ├── kustomization.yaml
│       │       └── kvm-dev-plg-ds.yaml
```

**Review by sherine-k — APPROVED** [review]
@lmzuccarelli 
great work! It finally runs!
One last nit pick in the README
Thanks

**Review by sherine-k — APPROVED** [review]

**Comment by lmzuccarelli** [comment]
> @lmzuccarelli to me it looks like this would break running the pipeline locally, as it removes the kvm device plugin DaemonSet. Could we instead sync what's here with what's in `apps` and just exclude those resources from the `operate-first` env overlay? (we could also create another env overlay for plain/new OpenShift clusters that also includes them, like in the local env, so people won't have to dig into `apps` to deploy all of this)

@LorbusChris 
I have re-added the daemonset for local Kind clusters (my bad). 
@sherine-k has retested using Kind locally with the updated changes (she reported that it took a total of 14 minutes). 
This means we need to ask for technical help regarding the kvm-device-plugin (myabe there are too many restrictions)

**Comment by LorbusChris** [comment]
LGTM now, but should we remove the old combined task too?

**Comment by lmzuccarelli** [comment]
@LorbusChris - Yes absolutely , I will update the PR and remove the old tasks.

**Comment by LorbusChris** [comment]
We can remove the old task later, let's merge this

**Pull request merged**